src/main/java/hello/controller/IssueController.java

```diff
@@ -0,0 +1,277 @@
+
+            OkHttpClient client = new OkHttpClient();
+
+            okhttp3.RequestBody body = okhttp3.RequestBody.create(JSON, json);
+            Request request = new Request.Builder().url(url).post(body).build();
+            Response response = client.newCall(request).execute();
+
+            return new Gson().fromJson(response.body().string(), CodeGenPostResponse.class);
+        } catch (IOException e) {
+            e.printStackTrace();
+            return new CodeGenPostResponse(-1);
+        }
+    }
+
+    private CodeGenGetResponse sendGetRequestCodeGen(String url) {
+        // send get request to codegen
+        try {
+            OkHttpClient client = new OkHttpClient();
+
+            Request request = new Request.Builder().url(url).build();
+            Response response = client.newCall(request).execute();
+
+            return new Gson().fromJson(response.body().string(), CodeGenGetResponse.class);
+        } catch (IOException e) {
+            e.printStackTrace();
+            return new CodeGenGetResponse("Failure", "");
+        }
+    }
+
+    private void fixIssue1(Integer methodId, String modifiers) {
+        String [] modifierList = {"public", "protected", "private", "static", "abstract", "synchronized", "transient", "volatile", "final", "native", "strictfp"};
+        String newModifiers = "";
+        for(String modifier : modifierList) {
+            if (modifiers.contains(modifier)) {
+                newModifiers += modifier + ", ";
+            }
+        }
+        if (newModifiers.endsWith(", ")) {
+            newModifiers = newModifiers.substring(0, newModifiers.length() - 2);
+        }
+        newModifiers = "[" + newModifiers + "]";
+        Session session = driver.session();
+        session.run("match (n) where id(n)=" + methodId + " set n.modifiers=\"" + newModifiers + "\"");
+    }
+
+    private void fixIssue3(Integer methodId, String modifiers) {
+        String [] modifierList = {"public", "protected", "private", "static", "abstract", "synchronized", "transient", "volatile", "final", "native", "strictfp"};
+        String newModifiers = "";
+        for(String modifier : modifierList) {
+            if (modifiers.contains(modifier) || modifier.equals("synchronized")) {
+                newModifiers += modifier + ", ";
+            }
+        }
+        if (newModifiers.endsWith(", ")) {
+            newModifiers = newModifiers.substring(0, newModifiers.length() - 2);
+        }
+        newModifiers = "[" + newModifiers + "]";
+        Session session = driver.session();
+        session.run("match (n) where id(n)=" + methodId + " set n.modifiers=\"" + newModifiers + "\"");
+    }
+
+    @GetMapping(path="/fix", produces = "application/json")
+    public String fixIssue(@RequestParam(value="issueId") Integer issueId, @RequestParam(value="sandBoxUrl") String sandBoxUrl) {
+        setNeo4jDriver(sandBoxUrl);
+        String USER = "neo4j";
```

Comment:
Not recommended to use constants directly in code. Define a utility class called Constants or something like that and define all your constants there. Usernames, queries, everything.